packages/calcite-components/src/components/handle/handle.scss

```diff
@@ -10,7 +22,7 @@
     self-stretch
     border-none
     bg-transparent;
-  color: theme("borderColor.color.input");
+  color: var(--calcite-color-border-input);
```

Comment:
I think this should be handled-text-color

Reply:
I think there needs to bet a var for text-color (border-input) and a var for hover/focus (text-color-1).